tests/test_flask_admin.py

```diff
@@ -82,6 +82,7 @@ def test_add_distro(self):
                 sess['email'] = 'pingou@pingoured.fr'
 
             output = c.get('/distro/add', follow_redirects=True)
+            print output.data
```

Comment:
Left over from testing indeed, but not a problem when ran with ``runtests.sh`` (cuts stdout unless the test breaks)